tests/unit_tests/pyanaconda_tests/__init__.py

```diff
@@ -112,21 +112,21 @@ def check_kickstart_interface(test, interface, ks_in, ks_out=None, ks_valid=True
     # Generate a kickstart
     ks_out = dedent(ks_out).strip()
     ks_generated = clear_version_from_kickstart_string(interface.GenerateKickstart()).strip()
-    test.assertEqual(ks_out, ks_generated)
+    assert ks_out == ks_generated
 
     # Test the properties changed callback.
     if ks_in is not None:
         callback.assert_any_call(KICKSTART_MODULE.interface_name, {'Kickstarted': True}, [])
     else:
-        test.assertEqual(interface.Kickstarted, False)
```

Comment:
Are we sure that we are not testing the False value here ? I mean, is value None acceptable ?

Reply:
Hmm, good point. Not sure really. I guess the same applies also for `True` value. It could also take anything.